cluster/images/etcd-version-monitor/etcd-version-monitor.go

```diff
@@ -191,7 +191,7 @@ func (m *monitorGatherer) rewriteExportedMetrics(metrics map[string]*dto.MetricF
 	return results, nil
 }
 
-// Struct for unmarshalling the json response from etcd's /version endpoint.
+// EtcdVersion struct for unmarshalling the json response from etcd's /version endpoint.
```

Comment:
It might be good to use wording "EtcdVersion is for ..." or something alike.

Reply:
Modified, please recheck, thanks!